app/build.gradle

```diff
@@ -180,7 +180,7 @@ dependencies {
 
     implementation 'me.zhanghai.android.materialprogressbar:library:1.6.1'
 
-    implementation 'net.jthink:jaudiotagger:3.0.1'
+    implementation 'org.bitbucket.drizztbsd:jaudiotagger:v3.0.1o'
```

Comment:
@drizzt Is it possible to update this to point to `https://github.com/VinylMusicPlayer/jaudiotagger` please?

Reply:
I'm doing that right now, sorry the delay but I'm at FOSDEM